server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/enums/TransportActionType.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.ratelimitting.admissioncontrol.enums;
+
+import java.util.Locale;
+
+/**
+ * Enums that defines the type of the transport requests
+ */
+public enum TransportActionType {
```

Comment:
Should this be a part of ActionModule where while defining actions the registry takes care of classifying

Reply:
Ack. This will be taken as part of followup PR.